src/lib/stateFulPassphraseForm.jsx

```diff
@@ -0,0 +1,108 @@
+/*
```

Comment:
This component is very specific to the passphrase form, it's why it's not the same than the statefulForm from myaccounts (which is specific to structures using a `fields` property like a connector). So, I don't think it's very useful to put those in cozy-ui as they can be very specific.